conbench/templates/benchmark-entity.html

```diff
@@ -165,6 +165,22 @@
           <a href="{{ run.commit.repository }}">{{ run.commit.display_repository }}</a>
         </div>
       </li>
+      {% if run.commit.branch %}
+      <li class="list-group-item" style="overflow-y: auto;">
+        <b>branch</b>
+        <div align="right" style="display:inline-block; float: right;">
+          {{ run.commit.branch }}
+        </div>
+      </li>
+      {% endif %}
+      {% if run.commit.fork_point_sha %}
+      <li class="list-group-item" style="overflow-y: auto;">
+        <b>forked from default at</b>
+        <div align="right" style="display:inline-block; float: right;">
+          {{ run.commit.fork_point_sha }}
```

Comment:
We might not yet have enough info to do this (especially on past commits), but it would be nice if we could construct URLs to these + ideally also use short hashes for display

Reply:
This is a bit of an extension of scope too, so if it's too much feel free to make an issue + defer it

Reply:
Yep see [this](https://github.com/conbench/conbench/pull/417#discussion_r1009559148) comment. I want to link to the commit like we link to the target commit, but I need to set up the backend in a future PR first